src/main/kotlin/com/recipebook/domain/ingredient/Ingredient.kt

```diff
@@ -0,0 +1,25 @@
+package com.recipebook.domain.ingredient
+
+import com.recipebook.domain.recipe.Recipe
+import com.recipebook.orm.AbstractJpaPersistable
+import java.io.Serializable
+import javax.persistence.Entity
+import javax.persistence.ManyToMany
+
+@Entity
+class Ingredient(var name: String, var unit: String, var quantity: Double) : AbstractJpaPersistable<Ingredient>(),
+        Serializable {
+    constructor() : this(name ="" ,unit = "", quantity = 0.00) {
+
+    }
```

Comment:
Nie potrzebujesz konstruktora tutaj.
Jeśli koniecznie chcesz defaultowe wartości to możesz zrobić np. 
`class Ingredient(var name: String = " ", var unit: String = " ", var quantity: Double = 0.0) `
Ale moim zdaniem jest to zbędne.
Tu znajdziesz więcej informacji o tym jak działają konstruktory w Kotlnie https://kotlinlang.org/docs/reference/classes.html

Reply:
Bez konstruktora wywala błąd, że klsa nie jest abstrakcyjna. Nie potrafię tego obejść

Reply:
Ok

Reply:
Bląd w klasie Ingredient - Nie wiem jak się go pozbyć.